integration-tests/kyuubi-hive-it/src/test/scala/org/apache/kyuubi/it/hive/HiveITMixin.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.kyuubi.it.hive
+
+import java.nio.file.Paths
+
+import org.apache.kyuubi.{KYUUBI_VERSION, SCALA_COMPILE_VERSION, Utils}
+
+trait HiveITMixin {
```

Comment:
let's remove it

Reply:
it's extended by two suites, removing it means duplicated code blocks

Reply:
hiveEngineBootstrap is not used anywhere, and kyuubiHome seems unnecessary for engine runtime

Reply:
removed